### ISSUE TITLE
Is it supposing to be taking this long to generate? (4060 ti 16gb vram)

### ISSUE THREAD

**Issue by scaruslooner** [ToonCrafter/ToonCrafter#28]
Its taking me a long time 12 mins to generate a video. I have a 4060ti 16gb vram gpu. Any suggests to make it go faster? 
![image](https://github.com/ToonCrafter/ToonCrafter/assets/39450507/535cb75a-37a9-4b02-80e8-1a0866aeecff)
![image](https://github.com/ToonCrafter/ToonCrafter/assets/39450507/1b7f7388-2794-402f-ad98-15332c7d0185)
![image](https://github.com/ToonCrafter/ToonCrafter/assets/39450507/a4db7fa2-c646-4575-a527-1bae47c1b0d2)
![image](https://github.com/ToonCrafter/ToonCrafter/assets/39450507/5edf9dca-2c17-4f5e-ac58-a9437396a10c)

**Comment by khawar-islam** [comment]
@scaruslooner I have the same problem and I am using 24GB VRAM but still i cannot generate video. It gives an error "CUDA out of memory" do you get this error? If not, did you change something?

**Comment by scaruslooner** [comment]
No, the only I did differently is rise a fork of tooncrafters called toocraft for windows and follow this video https://youtu.be/E89R5_hQ5bQ?si=ByQXA5TyXKCHJ64h

**Comment by slayeryahu** [comment]
A100 is required.

**Comment by zafirexd** [comment]
I use this implementation inside comfy ui https://github.com/kijai/ComfyUI-DynamiCrafterWrapper
![image](https://github.com/ToonCrafter/ToonCrafter/assets/100502318/b04ff86b-2286-4297-9563-b233d5cef516)
![image](https://github.com/ToonCrafter/ToonCrafter/assets/100502318/28eaedf3-c3de-45f5-b0d7-d165be8973b1)
Download this model. 
If you want to keep that environment replace all the mentions of the model.ckpt with the path of the fp16 path, in all of this files in your installation 
![image](https://github.com/ToonCrafter/ToonCrafter/assets/100502318/9bac479d-405b-45c4-baac-d3495ccab986)
i had a 3060 12gb and the max resolution i can use is 550x400 and 450x500, 11.8gb and the advantages for use inside comfy its the batch interpolation and use also rife interpolation, but it save all the previous frames in the vram, for 2 interpolations at the same time its the best, took from 1min to 3, but if start to use more than 11.9gb, goes brrrrr to 10 mins, 
![image](https://github.com/ToonCrafter/ToonCrafter/assets/100502318/f5d1688e-9516-417e-9479-d67efe863fea)
![image](https://github.com/ToonCrafter/ToonCrafter/assets/100502318/336a1d13-5123-4a62-b265

https://github.com/ToonCrafter/ToonCrafter/assets/100502318/f9209a59-617c-49ee-a190-b0d40c35f715

-18e08dfc2024)

**Comment by Utsa7** [comment]
@zafirexd thanks man! I installed comfyUI just for this. I have the same gpu so I ditto copied your workspace and it worked, but I am facing a problem, my generated clips are so short ( all 0 second) and I can't seem to make it to 2 seconds like you, do you have any idea how to achieve that? sorry I can't seem to understand most of the parameter there haha

**Comment by zafirexd** [comment]
@Utsa7 

Just decrease the frame rate in the video combine, Tooncraft only works properly with 16 frames, for 2 secs long video, change the frame rate in video combine, "frame_rate '8' "
![image](https://github.com/ToonCrafter/ToonCrafter/assets/100502318/7372b738-f050-45e2-b0d7-009281bd5f20)
